### PR TITLE
correct zone config privileges

### LINKED ISSUE #10999: Update docs regarding Admin and ZONECONFIG
**Is your feature request related to a problem? Please describe.**
You no longer require admin to update zone configs.

The note on this page is incorrect:
https://www.cockroachlabs.com/docs/stable/configure-replication-zones.html

image

**Describe the solution you'd like**
Delete the note or replace it.

**Describe alternatives you've considered**
A note that mentions something along the lines of:
"To allow users to manage their own zone configs, give their role the privilege ZONECONFIG" 

**Additional context**
Example of a non-admin updating zoneconfig:

`operator@:26257/defaultdb> SHOW RANGES from database defaultdb;
ERROR: only users with the admin role are allowed to read crdb_internal.ranges_no_leases
SQLSTATE: 42501
operator@:26257/defaultdb> SHOW ZONE CONFIGURATION FROM RANGE default;
     target     |              raw_config_sql
----------------+-------------------------------------------
  RANGE default | ALTER RANGE default CONFIGURE ZONE USING
                |     range_min_bytes = 134217728,
                |     range_max_bytes = 536870912,
                |     gc.ttlseconds = 90000,
                |     num_replicas = 3,
                |     constraints = '[]',
                |     lease_preferences = '[]'
(1 row)

Time: 2ms total (execution 2ms / network 0ms)

operator@:26257/defaultdb> ALTER TABLE defaultdb.test  CONFIGURE ZONE USING lease_preferences = '[[+region=local]]',constraints = '{"+region=local": 3, "+region=node3region": 2}', num_replicas=5;
CONFIGURE ZONE 1`

It looks like the feature was available from 20.1.0. Source:
https://github.com/cockroachdb/cockroach/blob/v20.1.0/pkg/sql/privilege/privilege.go

### PR DESCRIPTION
**Pull request by taroface**
Fixes #10999.

### CODE REVIEW

File: v20.1/configure-replication-zones.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ For each of the above objects you can control:
 This page explains how replication zones work and how to use the [`CONFIGURE ZONE`](configure-zone.html) statement to manage them.
 
 {{site.data.alerts.callout_info}}
-Currently, only members of the `admin` role can configure replication zones. By default, the `root` user belongs to the `admin` role.
+To configure replication zones, a user must be granted either the [`admin` role](authorization.html#admin-role), [`CREATE` privilege](authorization.html#supported-privileges), or [`ZONECONFIG` privilege](authorization.html#supported-privileges). To configure [`system` objects](#for-system-data), the `admin` role is required.
```
Comment (RichardJCai) [inline v20.1/configure-replication-zones.md:27]: I would reword this as `a user must be a member of the admin role`or have been granted ...

Same for the system part, would word it as a user must be a member of the admin role.

A user can be an indirect member of the admin role, ie, a member of a role that is a member of the admin role.

Ditto for the definitions below.

Comment (taroface) [reply v20.1/configure-replication-zones.md]: TFTR! Done.

**Review by RichardJCai — APPROVED** [review]

**Review by ianjevans — APPROVED** [review]
